python/private/versions.bzl

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The Python versions we use for the toolchains.
+"""
+
+RELEASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download/20211017"
```

Comment:
Have we established a line of comms with Gregory? I've been a very minor Github sponsor of him for a while, and have Twitter DM'd him, but taking this dependency does feel reminiscent of this XKCD comic: https://xkcd.com/2347/

I think at the least I'd like to get more hands on with this repo, and will message you on Slack in case I run into any speedbumps trying to compile these interpreters myself.

Reply:
I have not interacted with Gregory other than fixing a couple Make targets his repository uses to build OpenSSL.
Feel free to reach me on Slack.